Braze Demo Match Game Content Extension/Model/AttemptCounter.swift

```diff
@@ -0,0 +1,17 @@
+struct AttemptCounter {
+  private(set) var attempCount = 0
+  private var bestScore = Int.max
```

Comment:
I think it would be cleaner and safer to start with 0 and use a max function

Reply:
The `bestScore` is considered to be the lowest score as in lowest number of attempts it takes to match all the cards. Perhaps I should rename the variable to `lowScore`